Documentation/ceph-block.md

```diff
@@ -41,6 +41,9 @@ spec:
   failureDomain: host
   replicated:
     size: 3
+    # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
```

Comment:
Similar to the other comment, let's remove this setting from this main example to keep it simple